config.ru

```diff
@@ -4,9 +4,10 @@ require 'rack/ssl-enforcer'
 require 'travis/yml/web'
 
 env = Travis::Yml::Web::Env
+config ||= Travis::Yml::Web::Config.load
```

Comment:
I think this is already loaded here https://github.com/travis-ci/travis-yml/blob/master/lib/travis/yml/web/basic_auth.rb#L18. So it could be accessed without loading, but I can not find an example.